config/embed.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+import _ "embed"
+
+var (
+	//go:embed crd/bases/core.arlon.io_callhomeconfigs.yaml
```

Comment:
We have to remember to add new CRDs here e.g. upcoming AppProfile

Reply:
Will add an issue

Reply:
https://github.com/arlonproj/arlon/issues/301